### DISCUSSION TITLE
I'm a convert

### DISCUSSION THREAD

**Discussion by corysimmons** [tailwindlabs/tailwindcss#8659] [Kind words]
I tried Tailwind a few years ago; didn't like it because it bloated my markup; and even warned people against it in favor of just mastering CSS (I still think people should do this anyway).

I gave it another shot with TailwindUI; accessibility is built in; componentized everything so even if there are a lot of classes they are quarantined to small components; realized how much I appreciate pre-built components that I can easily tweak to suit my needs; absolutely love the pre-packaged design system (the hand-picked colors are so nice).

It's just a huge time saver and the design is really :chef_kiss:

Headless UI is just an objectively great idea with great execution as well. I'd love to see more work in that space (tables with all the react-table goodness, etc.).

Types, etc... just a ton of quality work put into this...

Great job, Tailwind team!

**Comment by RobinMalfait** [comment]
Hey, thanks for the kind words!

Welcome 😀
